tests/k6/components/utils.js

```diff
@@ -33,7 +34,7 @@ export function setupBase(config) {
 
                 ctl.loadModelFn(modelName, defs.model.modelDefn, true)
                 if (config.isLoadPipeline) {
-                    ctl.loadPipelineFn(generatePipelineName(modelName), defs.model.pipelineDefn, false)  // we use pipeline name as model name + "-pipeline"
```

Comment:
We now wait for Pipelines to be marked as ready; Otherwise, we might get spurious state inconsistencies (i.e a pipeline becoming ready during the check)